cockatrice/src/carditem.cpp

```diff
@@ -110,7 +110,7 @@ void CardItem::paint(QPainter *painter, const QStyleOptionGraphicsItem *option,
         QStringList ptDbSplit = db->getCard(name)->getPowTough().split("/");
         QStringList ptSplit = pt.split("/");
         
-        if (ptDbSplit.at(0) != ptSplit.at(0) || ptDbSplit.at(1) != ptSplit.at(1))
+        if ((ptDbSplit.at(0) != ptSplit.at(0) || ptDbSplit.at(1) != ptSplit.at(1)) && !getFaceDown())
```

Comment:
Change to: 
`if (facedown || ptDbSplit.at(0) != ptSplit.at(0) || ptDbSplit.at(1) != ptSplit.at(1))`

If the card is face down it should always be orange. Also performs this checks first so we dont waste cycles on comparisons if not needed.